test/wasm-js/wasm-js.status

```diff
@@ -25,8 +25,8 @@
   'proposals/reference-types/limits': [SKIP],
   'proposals/bulk-memory-operations/limits': [SKIP],
   'proposals/js-types/limits': [SKIP],
-  # TODO(wasm): Update memory limit.
-  'limits': [FAIL],
+  'proposals/simd/limits': [SKIP],
```

Comment:
Should us skip only when arch is riscv64?

Reply:
Sure, it should be skipped only for RV64.
Let's just  go ahead for this commit to see whether it can pass CI.
Then add the constraint.